content/learn.html

```diff
@@ -13,10 +13,10 @@
   <div class="container">
     <div class="hero-area hero-area-row row">
       <h1 class="hero-title no-link">Learn</h1>
-      <p>Explore our structured learning paths to discover everything you need to know about building for the modern web.</p>
+      <p>For each topic, learn best practices for the modern web and hone your skills with a hands-on codelab.</p>
```

Comment:
technically not every guide has an accompanying code lab. Maybe that's ok though?

Reply:
No, that's good feedback, because this sentence says "each topic." I can rephrase as something like "Across several topics..."

Reply:
Sure, if you want to send me the new text line you'd like to include I can probably update the pull request with it.

Reply:
Oh, thanks! I was trying to edit myself and couldn't figure out how :) Can you please update to just "Learn best practices for the modern web and hone your skills with hands-on codelabs." This string avoids the "each topic" inaccuracy, and it's already clear that the learning plans & codelabs are organized by topic because they're right there, under the string.